contracts/test/migration/TLC_globalUnlockScheduleMigration.t.sol

```diff
@@ -47,69 +153,29 @@ contract TlcMigrationTest is Test {
 
         tlc = new TLCV1();
         vm.prank(0x0D1dE267015a75F5069fD1c9ed382210B3002cEb);
-        tlcProxy.upgradeTo(address(tlc));
+        tlcProxy.upgradeTo(address(0xF8745c392feF5c91fa1cdB0202efF7Ca08dF55ce));
 
-        assertEq(tlcProxy.getVestingScheduleCount(), 67);
+        assertEq(tlcProxy.getVestingScheduleCount(), 103);
 
         // Check that the all the values that shouldn't change didn't
-        for (uint256 i = 0; i < 67; i++) {
+        for (uint256 i = 0; i < tlcProxy.getVestingScheduleCount(); i++) {
             VestingSchedulesV2.VestingSchedule memory schedule = TLCV1(address(tlcProxy)).getVestingSchedule(i);
+            assertEq(schedule.start, schedulesBefore[i].start);
+            assertEq(schedule.end, schedulesBefore[i].end);
+            assertEq(schedule.cliffDuration, schedulesBefore[i].cliffDuration);
+            assertEq(schedule.duration, schedulesBefore[i].duration);
+            assertEq(schedule.periodDuration, schedulesBefore[i].periodDuration);
             assertEq(schedule.amount, schedulesBefore[i].amount);
             assertEq(schedule.creator, schedulesBefore[i].creator);
             assertEq(schedule.beneficiary, schedulesBefore[i].beneficiary);
             assertEq(schedule.revocable, schedulesBefore[i].revocable);
             assertEq(schedule.releasedAmount, schedulesBefore[i].releasedAmount);
         }
```

Comment:
Hey @iamsahu I noticed you didn't check to confirm ignoreGlobalUnlock shouldn't change, is this not necessary?